charts/.ci/ct-config.yaml

```diff
@@ -1,4 +1,5 @@
 # This file defines the config for "ct" (chart tester) used by the helm linting GitHub workflow
+all: true
```

Comment:
Our `AutoScalingRunnerSet` depends on the `actions-runner-controller-2` chart being installed.
The CT will try to install charts that only have changes by default which means the `AutoScalingRunnerSet` will fail to install since the CRDs are not there.
I am updating the `ct-config` to install all charts by default.